scripts/downloader.py

```diff
@@ -0,0 +1,77 @@
+import asyncio
+import os
+
+from requests import get
+
+
+DOWNLOAD_DIR = './_downloaded'
+PROJECT_URL = 'https://circleci.com/api/v1.1/project/github/honzajavorek/junior.guru'
+
+
+def is_fetch_job(job):
+    return job.get('workflows', {}).get('job_name', '') == 'fetch'
+
+
+def is_nightly_job(job):
+    return job.get('workflows', {}).get('workflow_name', '') == 'nightly'
+
+
+def is_fetch_and_nightly(job):
+    return is_fetch_job(job) and is_nightly_job(job)
+
+
+def fetch_url(build_num):
+    try:
+        res = get(f'{PROJECT_URL}/{build_num}/artifacts')
+        artifacts = list(filter(lambda artifact: artifact.get('path', '') == 'backup.tar.gz', res.json())) 
+        # only one artifact should be there
+        artifact = artifacts[0] if len(artifacts) > 0 else {}
+
+        return artifact.get('url', None)
+    except Exception as e:
+        # something has failed, but no big deal
```

Comment:
Nemělo by to spíš spadnout, když se to nepovede stáhnout? 🤔

Reply:
Spíš ne, když se nepovede stáhnout jeden, tak to IMHO nevadí.

Reply:
OK